app/src/main/java/org/mozilla/focus/ui/theme/FocusTypography.kt

```diff
@@ -79,6 +80,11 @@ val focusTypography: FocusTypography
             fontSize = 20.sp,
             color = focusColors.onPrimary,
         ),
+        dialogContent = TextStyle(
```

Comment:
Just want to point out that adding colors and typography styles for every new widget is not sustainable going forward. Looks like Focus could benefit from the approach used on Fenix where we have only a small subset of styles and colors that everybody strives to use. This should be discussed with UX.